### PR TITLE
Add Lite version with Ads.

### PR DESCRIPTION
**Pull request by Metaxa007**

### CODE REVIEW

File: OpenDocumentReader/ConfigurationManager.swift [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+struct ConfigurationManager {
+    static let manager = ConfigurationManager()
+    
+    private(set) var configuration: String!
```
[the remark this answers is not in the document]
Comment (Metaxa007) [reply OpenDocumentReader/ConfigurationManager.swift:6]: only setter is private. getter is public

[the remark this answers is not in the document]
Comment (Metaxa007) [reply OpenDocumentReader/ConfigurationManager.swift:6]: no, we can not control the background color. There is a property adSize, but the documentation says, we should not create own sizes, but use predefined that can not cover the whole width. Probably i need to investigate more how to fix this problem.

**Comment by TomTasche** [comment]
are we using adaptive banners? those should fill the screen as much as possible https://developers.google.com/admob/ios/banner/adaptive

File: OpenDocumentReader.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -12,6 +12,8 @@
 		AC384BCD23B4FFA700C7BF47 /* ContentViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = AC384BC923B4FFA700C7BF47 /* ContentViewController.swift */; };
 		AC384BCE23B4FFA700C7BF47 /* Constants.swift in Sources */ = {isa = PBXBuildFile; fileRef = AC384BCA23B4FFA700C7BF47 /* Constants.swift */; };
 		AC384BCF23B4FFA700C7BF47 /* PageViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = AC384BCB23B4FFA700C7BF47 /* PageViewController.swift */; };
+		AC6A1CD9243E6B8400CD42EC /* (null) in Sources */ = {isa = PBXBuildFile; };
```
Comment (TomTasche) [inline OpenDocumentReader.xcodeproj/project.pbxproj:15]: what's that? 🤷

Comment (Metaxa007) [reply OpenDocumentReader.xcodeproj/project.pbxproj:15]: weird, i recreated both configurations from scratch. here ist the that i used: [guide](https://www.youtube.com/watch?v=r242pYLr-Gg&t=)

**Comment by TomTasche** [comment]
this seems like a good approach: https://blog.testfairy.com/2-ios-apps-1-codebase-with-xcode-scheme/

problem with yours right now: even while developing you are building the app in release mode, which means longer build times and worse debugging support! @Metaxa007

**Comment by Metaxa007** [comment]
@TomTasche thanks, nice tutorial. I implemented changes according to it.

**Comment by TomTasche** [comment]
Looks very good now! I'll do some more tests with uploading to the App Store to make sure it's working.

Would it be possible to hide / show the ad-area based on if an ad was loaded? Two scenarios I can think of:
1. User starts the app and opens a document: white area is shown at the top until ad is loaded
2. If the user doesn't have internet connectivity or no ad was loaded for another reason, the user will continue to see a white area

**Comment by TomTasche** [comment]
Making a few commits to this branch to resolve merge conflicts. Please pull before making any further changes

**Comment by Metaxa007** [comment]
I'll take a look. If it has a completion handler, would be very easy to implement what you described.

**Comment by Metaxa007** [comment]
1. "User starts the app and opens a document: white area is shown at the top until ad is loaded" to avoid it i have to set the view height to 0 and hide the view for the beginning, but in this case delegate immediately throws `didFailToReceiveAdWithError` and the Ad can not be loaded and shown.
2. "If the user doesn't have internet connectivity or no ad was loaded for another reason, the user will continue to see a white area". That is where `didFailToReceiveAdWithError` is called and the view can be hidden.

**Pull request merged**